scripts/nginx.sh

```diff
@@ -284,6 +284,20 @@ function set_nginx_cfg() {
   fi
 }
 
+function set_nginx_permissions() {
+  local distro_name version_id
+
+  distro_name=$(grep -E "^NAME=" /etc/os-release | cut -d'"' -f2)
+  version_id=$(grep -E "^VERSION_ID=" /etc/os-release | cut -d'"' -f2)
+
+  if [[ ${distro_name} == "Ubuntu" && ( ${version_id} == "21.10" || ${version_id} == "22.04") ]]; then
```

Comment:
This will require attention again in October when 22.10 comes out. Unless this change is harmful on 21.04 it may be better to just remove the version check altogether.

Perhaps the script could simply check if the nginx user has sufficient permission and run chmod if it does not? Unsure if things like selinux or other sandboxing may come into play here.

Reply:
Fair point. Maybe not a bad idea indeed. Or at least something like checking if `distro_name == "Ubuntu"` and if that is the case i then check for the required permissions. Another way would be to check for major version greater than `20`. Because up until Ubuntu 20, no permission changes are required.